src/autoscaler/api/brokerserver/broker_handler.go

```diff
@@ -117,7 +265,7 @@ func (h *BrokerHandler) DeleteServiceInstance(w http.ResponseWriter, r *http.Req
 	if err != nil {
 		if err == db.ErrDoesNotExist {
 			h.logger.Error("failed to delete service instance: service instance does not exist", err,
-				lager.Data{"instanaceId": instanceId})
+				lager.Data{"instanceId": instanceId})
```

Comment:
when re-reading the `deleteserviceinstance` implementation, I just realized that when a default policy exists,  if we just remove the service instance, then an un-used policy will be left behind in the policy db forever.  Is my understanding correct?  @silvestre ?

Reply:
No, you cannot remove a service instance without deleting all bindings first.